### PR TITLE
Some small tweaks/suggestions for the method section

### PR DESCRIPTION
**Pull request by seberg**
Mainly small teaks, I will make a few inline comments as well. I did not read the wheels and testing section super carefully.

### CODE REVIEW

File: methods.tex
Comment (seberg) [inline methods.tex]: I assume we were talking about distutils here? I tried to split things up a bit, not sure it was fully successfull.

File: methods.tex
Comment (seberg) [inline methods.tex:353]: I stumbled over "deviates" a bit, probably just me, but variates was used all over.

File: methods.tex
Comment (seberg) [inline methods.tex]: Was just curious if we want to keep it so specific (it is fully correct now of course). I am happy with it as is though.

File: methods.tex
Comment (seberg) [inline methods.tex:32]: not sure it makes a difference, I stumbled a bit because a sliced array itself is not contiguous anymore, of course it still  is a "view" into a contiguous block.

File: methods.tex
Comment (seberg) [inline methods.tex:46]: I assume it is OK to reference images in the main article?

File: methods.tex
Comment (seberg) [inline methods.tex]:
```suggestion
are implemented as ufuncs, so broadcasting applies even in expressions
```
stumbled over "even", but rereading "also" is too weak...

**Comment by jarrodmillman** [comment]
This all seems reasonable to me.  I will merge later tonight unless you complain.

File: methods.tex
Comment (stefanv) [reply methods.tex:32]: Good point!

File: methods.tex
Comment (stefanv) [reply methods.tex]: How about "so that broadcasting applies also to"?

File: methods.tex
Comment (stefanv) [reply methods.tex]: "to as part" -> ...

Although, I'm not sure our customizations of distutils have much to do with PyPi?

File: methods.tex
Comment (stefanv) [inline methods.tex]: This means that, for most decisions, everyone is...

File: methods.tex
Comment (stefanv) [reply methods.tex]: "a small amount" may suffice; it's a small enough amount that it probably doesn't matter either way :)

File: methods.tex
Comment (stefanv) [reply methods.tex:353]: deviates should probably used throughout; not sure how that slipped in there!

File: methods.tex
Comment (stefanv) [inline methods.tex]: a key feature of a seed sequence, perhaps?

File: methods.tex
Comment (stefanv) [inline methods.tex]: What does distinct mean here?  Do you think we need to be more precise, or is that enough?

**Review by stefanv — APPROVED** [review]
Good edits, thanks!  I made a few minor suggestions, but we can also apply those after merging.

File: methods.tex
Comment (jarrodmillman) [reply methods.tex:353]: I think "variates" is correct, no?

> A random variate is a particular outcome of a random variable: the random variates which are other outcomes of the same random variable might have different values. A random deviate or simply deviate is the difference of random variate with respect to the distribution central location (e.g., mean), often divided by the standard deviation of the distribution.

- https://en.wikipedia.org/wiki/Random_variate

File: methods.tex
Comment (jarrodmillman) [reply methods.tex]: Let's go with small amount.

**Comment by jarrodmillman** [comment]
Since there are some good suggestions, I will wait to merge this.

File: methods.tex
Comment (seberg) [reply methods.tex]: Good one, just realized, we also really need to cite Professor O'Neils work here if we did not already. She probably has some nice phrase for distinct, although the opposite direction to just tell "its safe to use it to create as many generators as you wish" may be just as well.

File: methods.tex
Comment (seberg) [reply methods.tex]: Sounds good. The even does not really fit here I think, but it had the nice ring of being deeply ingrained into everything numpy does.

File: methods.tex
Comment (seberg) [reply methods.tex]: If this feels too strong, we could add the word "initially", or similar.

**Pull request merged**
